plugins/main/sriov/README.md

```diff
@@ -0,0 +1,54 @@
+# SR-IOV CNI plugin
+
```

Comment:
It seems a bit empty here! Could you add an introduction to this plugin that explains what it does and what it's used for?